packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -568,7 +568,14 @@ export class RunningSummarizer extends TypedEventEmitter<ISummarizerEvents> impl
 				};
 				const summarizeResult = this.generator.summarize(summaryOptions, resultsBuilder);
 				// ensure we wait till the end of the process
-				return summarizeResult.receivedSummaryAckOrNack;
+				const result = await summarizeResult.receivedSummaryAckOrNack;
+				if (!result.success) {
+					this.mc.logger.sendErrorEvent({
```

Comment:
Haven't looked at the code but I was wondering if we can tell/save what was the error for it.